CalibCalorimetry/HcalAlgos/src/HcalDbASCIIIO.cc

```diff
@@ -357,34 +332,36 @@ bool dumpHcalMatrixObject (std::ostream& fOutput, const T& fObject) {
 
 // ------------------------------ end templates ------------------------------
 
-bool HcalDbASCIIIO::getObject (std::istream& fInput, HcalGains* fObject) {return getHcalObject (fInput, fObject, new HcalGain);}
-bool HcalDbASCIIIO::dumpObject (std::ostream& fOutput, const HcalGains& fObject) {return dumpHcalObject (fOutput, fObject);}
-bool HcalDbASCIIIO::getObject (std::istream& fInput, HcalGainWidths* fObject) {return getHcalObject (fInput, fObject, new HcalGainWidth);}
-bool HcalDbASCIIIO::dumpObject (std::ostream& fOutput, const HcalGainWidths& fObject) {return dumpHcalObject (fOutput, fObject);}
+namespace HcalDbASCIIIO {
+    bool getObject (std::istream& fInput, HcalGains* fObject) {return getHcalObject<HcalGain> (fInput, fObject);}
```

Comment:
I'm not sure about the logic to fold implementations of these methods (declared in CalibCalorimetry/HcalAlgos/interface/HcalDbASCIIIO.h) in a namespace, while a few other methods implemented below are left with the namespace prepended.

Reply:
The other functions are spread out so far that I thought putting them in the namespace reduces readability. Since the functions in this block are just multiple one liners, I thought putting them in a namespace is fine to make the code less verbose. While it can be still easily understood that they are in the namespace.

But in the end I have no strong opinion on this and could write `HcalDbASCIIIO::` for each function.

Reply:
ok